packages/manager/modules/cdn-dedicated/src/dedicated/domain/add/index.js

```diff
@@ -12,7 +12,10 @@ angular
   .controller('CdnAddDomainsCtrl', controller)
   .run(
     /* @ngInject */ ($templateCache) => {
-      $templateCache.put('domain/add/cdn-dedicated-domain-add.html', template);
+      $templateCache.put(
+        'cdn/dedicated/domain/add/cdn-dedicated-domain-add.html',
```

Comment:
No need to update path where it the template is called ?

Reply:
There was something strange before. Even if the template wasn't added into template cache, it worked as expected. Since this code has moved into modules, it didn't work anymore and I don't know why. Maybe a webpack process that add the template?
As in the `$scope.setAction()` method, there is `cdn/dedicated`, I have added it in all `$templateCache.put`.